index.html

```diff
@@ -376,22 +376,23 @@ <h3>
               </p>
               <p class="draft-status">
                 <b>Draft state:</b> <a href=
-                "https://www.w3.org/TR/2017/CR-presentation-api-20170601/">CR</a> / Stable
+                "https://www.w3.org/TR/2023/CRD-presentation-api-20231016/"><abbr title=
+                "Candidate Recommendation Draft">CRD</abbr></a>
               </p>
               <p class="milestone">
-                <b>Expected completion:</b> Q4 2023 &mdash; The Working Group is awaiting
+                <b>Expected completion:</b> Q4 2025 &mdash; The Working Group is awaiting
                 satisfaction of the <a href="#success-criteria">Success Criteria</a>, which require
                 two interoperable implementations of the Presentation API. The Working Group will
                 also await demonstration of interoperability among multiple browsers and multiple
                 presentation displays before moving the specification to Proposed Recommendation.
               </p>
               <p>
                 <b>Reference Draft:</b> <a href=
-                "https://www.w3.org/TR/2020/CRD-presentation-api-20201105/">https://www.w3.org/TR/2020/CRD-presentation-api-20201105/</a><br>
+                "https://www.w3.org/TR/2017/CR-presentation-api-20170601/">https://www.w3.org/TR/2017/CR-presentation-api-20170601/</a><br>
```

Comment:
Why is this link changed to an older version of the spec?

Reply:
The current version is here: https://www.w3.org/TR/2023/CRD-presentation-api-20231016/

Reply:
Yes, the link in the previous charter was wrong ;) The "Reference Draft" is meant to be what the Process calls the [Exclusion Draft](https://www.w3.org/2023/Process-20231103/#exclusion-draft), in other words, the last published version that triggered a call for exclusions. Candidate Recommendation Drafts do not trigger calls for exclusions, only Candidate Recommendation Snapshots do.

Thinking about it, I'm going to refresh the list of deliverables to use the terms defined in the Process. The charter uses different terms only because it predates the appareance of these definitions in the Process. It seems a good idea to align things.